frontend/src/App.tsx

```diff
@@ -105,6 +112,11 @@ const App = (): React.ReactElement => {
                   path={Routes.PREVIEW_REVIEW_TEST}
                   component={PreviewReviewTest}
                 />
+                <Route
```

Comment:
Could you add a TODO to properly address this later? It's fine for now, but I think best practice is to actually swap components instead of redirecting

Reply:
done